### ISSUE TITLE
How to display the Custom cast column?

### ISSUE THREAD

**Issue by sd-evo** [open-admin-org/open-admin#152]
I have custom cast class, which return MoneyPhp object. Laravel doc: https://laravel.com/docs/10.x/eloquent-mutators#custom-casts
My code:
```php
$grid->column('current_balance', __('admin.grid.wallet.current_balance'))->display(function ($value) {
    $decimalFormatter = new DecimalMoneyFormatter(new ISOCurrencies());

    return $decimalFormatter->format($value);
});
```

I get error:

![Снимок экрана от 2024-02-05 03-19-28](https://github.com/open-admin-org/open-admin/assets/66308935/644dfa47-d374-4f34-8068-b716de60a19b)

How to display the column which using Custom cast?

**Comment by RCyrix** [comment]
cast it as a string.. it expects a string value.

**Comment by sd-evo** [comment]
> cast it as a string.. it expects a string value.

Thanks for answer!

But I specifically use Custom cast to MoneyPhp in the application. I have app and open admin panel. I can't to use the string type.
Any ideas?

**Comment by RCyrix** [comment]
grid is only display, it doesnt submit data, only "views" data, the form would handle data submission

**Comment by sd-evo** [comment]
> grid is only display, it doesnt submit data, only "views" data, the form would handle data submission

Yes, but grid expects that the field "current_balance" is string... How I can show this data type(MoneyPhp) in grid using method "display"? Strict typing is the best practice, so I use custom cast to MoneyPhp in Eloquent.

**Comment by RCyrix** [comment]
I am unable to assist here.
